src/components/GoCookieConsent/GoCookieConsent.vue

```diff
@@ -153,13 +246,56 @@ export default {
         support our marketing efforts. Click here to choose the type of cookies that are
         setup on your device.`
   const labelAccept = 'Accepter'
+  const model = {
+    cookie_monitoring: true,
+    cookie_media: true,
+    cookie_marketing: true,
+  }
+  const inputs = [
+    {
+      "type": "checkbox",
+      "inputName": "cookie_monitoring",
+      "label": "Audiance monitoring",
+      "id": "cookie_monitoring",
+      "model": "cookie_monitoring",
+      "wrapperClasses": "text-white is-inverse"
+    },
+    {
+      "type": "checkbox",
+      "inputName": "cookie_media",
+      "label": "Social media",
+      "id": "cookie_media",
+      "model": "cookie_media",
+      "wrapperClasses": "text-white is-inverse"
+    },
+    {
+      "type": "checkbox",
+      "inputName": "cookie_marketing",
+      "label": "Marketing",
+      "id": "cookie_marketing",
+      "model": "cookie_marketing",
+      "wrapperClasses": "text-white is-inverse"
+    },
+  ]
+  let isOpenList = false
 
   <div>
+    <button class="btn btn-sm btn-primary m-b-xl" @click="isOpenList = !isOpenList">OpenList</button>
+    <div
+      v-for="(value, name, key) in model"
+      :key="key"
+    >
+      {{ name }}: {{ value }}
+    </div>
     <go-cookie-consent
       :textContent="text"
       :acceptLabel="labelAccept"
       :privacyData="{url: 'master.agorize.com', label: 'Our privacy content'}"
```

Comment:
We shouldn't have static urls

Reply:
This is only for documentation @MatthieuGC